cmd/tarkov-tk-bot/serve.go

```diff
@@ -42,9 +40,102 @@ func serveCmd() *cobra.Command {
 	}
 }
 
+func init() {
+	ctx := context.Background()
+	s, err := firestore.NewKillStore(ctx, "tarkov-tk-dev")
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	store = s
```

Comment:
sorry, there's actually no point putting this in an init, it can just go in serveRun. Remember to defer s.Close()